operator-wg/whitepaper/11_Related_Work.md

```diff
@@ -1,13 +1,23 @@
 **Status**: WIP | **Maintainer** : N/A | 
 
 
-## Related Work / What’s next? / Where to get help?
-**(Current) Issue: https://github.com/cncf/sig-app-delivery/issues/47**
-
+## Related Work
 Initially, Operators were introduced by a blog post on the CoreOS Blog (Philips, 2016) . This article provides a rough overview what operators are, why the concept has been developed and how they are built. The insights of this article are mainly used for the definition of operators in this document. As the blog post only provided a concise overview, additional terms as capabilities, security and additional concepts are described more in-depth in this document.
 
+Ref: Philips, B. (2016, 03 16). Introducing Operators: Putting Operational Knowledge into Software. CoreOS Blog. Retrieved 11 24, 2020, from https://coreos.com/blog/introducing-operators.html
```

Comment:
> https://coreos.com/blog/introducing-operators.html
When I open this link I am directed to https://www.openshift.com/blog which is not as helpful for people who will be reading/trying to learn about Operators.

Reply:
You're right, the target seems to be removed ... Should we remove the link or leave this as a reference (as there is also a retrieval date)?

Reply:
I think we should keep things current. Either find another link where this is quoted or remove the link from the text.